src/Messaging/src/RabbitMQ/Support/RabbitMessageBuilder.save

```diff
@@ -0,0 +1,703 @@
+﻿// Copyright 2017 the original author or authors.
```

Comment:
Why do we need to keep this file in source?

Reply:
We don't .. deleted